wpilibj/src/athena/java/edu/wpi/first/wpilibj/HardwareDriverStation.java

```diff
@@ -78,23 +68,14 @@ public void run() {
   private final long m_packetDataAvailableSem;
 
   /**
-   * Gets an instance of the DriverStation
-   *
-   * @return The DriverStation.
```

Comment:
Is the idea here for documentation to be inherited? I didn't realize that worked.